report_csv/controllers/main.py

```diff
@@ -94,13 +94,13 @@ def report_download(self, data, context=None):
                         report_name = safe_eval(
                             report.print_report_name, {"object": obj, "time": time}
                         )
-                        filename = "%s.%s" % (report_name, "csv")
+                        filename = f"{report_name}.csv"
                 response.headers.add(
                     "Content-Disposition", content_disposition(filename)
                 )
                 return response
             else:
-                return super().report_download(data, context)
+                return super().report_download(data, context, token)
```

Comment:
Always keep keyword arguments:

```suggestion
                return super().report_download(data, context, token=token)
```